src/crypto/transform.py

```diff
@@ -10,19 +10,8 @@
 from google.cloud.storage.blob import Blob
 
 from crypto.utils.helpers import service_credentials
-from crypto.utils.constants import (
-    BUCKET,
-    CLOUD_STORAGE,
-    BIGQUERY,
-    PROJECT_ID,
-    DAY_DIM,
-    MONTH_DIM,
-    DATE_DIM,
-    INITIAL_LOAD,
-    NAME_DIM,
-)
-from crypto.utils.setup import DayDim, MonthDim, DateDim, NameDim
-
+from crypto.utils.constants import Project, Table
+from crypto.utils.setup import *
```

Comment:
Knew I shouldn't of done this :) I know it's not best practice. My thinking was since small and main/only use of `setup` was to house namedtuples then it might be okay. I didn't like how many namedtuples I was importing. Another thought I had was to do `from crypto.utils import setup` and then reference as `setup.<namedtuple>`. Didn't like that though, using the naming `setup`. I saw your other comment about changing the module name so that might be a solution.

Reply:
Nws, that's what the reviewing is for :)
Creating a namespace like that with setup.VAR is nice too 🎉

Reply:
Thanks Bob. That's a really good suggestion about setup.VAR too.